pandas/tests/arithmetic/test_interval.py

```diff
@@ -133,7 +133,7 @@ def test_compare_scalar_na(self, op, array, nulls_fixture, request):
         result = op(array, nulls_fixture)
         expected = self.elementwise_comparison(op, array, nulls_fixture)
 
-        if nulls_fixture is pd.NA and array.dtype != pd.IntervalDtype("int64"):
+        if nulls_fixture is pd.NA and array.dtype != IntervalDtype("int64"):
```

Comment:
can you revert this change? Only `pd.Interval` needed changing, not `pd.IntervalDtype`